raiden/network/transport/udp/healthcheck.py

```diff
@@ -166,3 +165,7 @@ def healthcheck(
                 )
                 event_unhealthy.clear()
                 event_healthy.set()
+
+
+if TYPE_CHECKING:
+    from raiden.network.transport.udp.udp_transport import UDPTransport
```

Comment:
If i understand correctly, this is only to satisfy the "existence" of `UDPTransport` definition in this file?

Reply:
This should fix our circular dependency (healtcheck uses the transport, and the transport uses healhcheck), and allow mypy to do the proper type checking (otherwise it would not know what is the actual interface of `RaidenService` and all call sites fail to type check)